README.md

```diff
@@ -47,30 +47,28 @@ If you find bugs or have problems with the CalDAV-Backend, you should ask the te
 
 ## Build the app
 
-Make sure you have all the [necessary prerequisites](https://docs.nextcloud.com/server/latest/admin_manual/installation/source_installation.html#prerequisites-for-manual-installation) for manual installs
+General build instructions for the app itself are the same for all options.
```

Comment:
**question:** What does `all options` mean here?

Reply:
I don't understand your suggestion nor what `all option` means here. That is not what is in the `README.md` file in my pull-request.

Reply:
@tcitworld I was looking at the wrong part of the README due to github coloring. I had copied the block from Deck app and the `all options` line was an oversight.

Apologies for confusion, thanks for catching my mistake 😄